cafeteria/src/main/java/org/itt/utility/LoggerUtility.java

```diff
@@ -0,0 +1,22 @@
+package org.itt.utility;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LoggerUtility {
+    private static final String LOG_FILE_PATH = "user_login_logout_log.txt";
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+    public static void log(String message) {
```

Comment:
Does this need to be static? Please go through your code and remove static functions unless really needed.